lib/include/virtqueue.h

```diff
@@ -41,16 +41,16 @@ typedef uint8_t boolean;
 
 /*Error Codes*/
 #define VQ_ERROR_BASE            (-3000)
-#define ERROR_VRING_FULL         (VQ_ERROR_BASE - 1)
-#define ERROR_INVLD_DESC_IDX     (VQ_ERROR_BASE - 2)
-#define ERROR_EMPTY_RING         (VQ_ERROR_BASE - 3)
-#define ERROR_NO_MEM             (VQ_ERROR_BASE - 4)
-#define ERROR_VRING_MAX_DESC     (VQ_ERROR_BASE - 5)
-#define ERROR_VRING_ALIGN        (VQ_ERROR_BASE - 6)
-#define ERROR_VRING_NO_BUFF      (VQ_ERROR_BASE - 7)
-#define ERROR_VQUEUE_INVLD_PARAM (VQ_ERROR_BASE - 8)
-
-#define VQUEUE_SUCCESS (0)
+#define ERROR_VRING_FULL         (VQ_ERROR_BASE - 1U)
```

Comment:
VQ_ERROR_BASE is signed so why it is needed to change the 1 to 1U? What MISRA rule is broken here?

Reply:
Hmm this looks like wrong identification. Sorry. will revert

Reply:
ok, thanks